egs/wsj/s5/conf/segment_long_utts.conf

```diff
@@ -0,0 +1,28 @@
+# TF-IDF similarity search options
```

Comment:
Putting so many options in a config file like this is, IMO, not a good idea because it's bewildering to users.
Can't you just set all these as the defaults in whatever script this is consumed by?
Why does this even needs its own config file?